internal/icinga2/client.go

```diff
@@ -441,6 +555,14 @@ func (client *Client) Process() {
 	client.eventDispatcherEventStream = make(chan *eventMsg)
 	client.catchupPhaseRequest = make(chan struct{})
 
+	cache, err := lru.New[string, map[string]string](int(math.Exp2(17)))
```

Comment:
```suggestion
	cache, err := lru.New[string, map[string]string](1 << 17)
```

Doing so will also get rid of the `math` import.